src/extension/background-script/actions/allowances/delete.ts

```diff
@@ -0,0 +1,12 @@
+import type { MessageAllowanceDelete } from "~/types";
+
+import db from "../../db";
+
+const deleteAllowance = async (message: MessageAllowanceDelete) => {
+  const id = message.args.id;
+  await db.allowances.delete(parseInt(id));
```

Comment:
Allowances `id` per type definitions are `string` but here we need to switch it to a `number`? Not sure if I understand.

Reply:
Maybe have a look at this: https://github.com/getAlby/lightning-browser-extension/pull/988/files#diff-1b329da99ce24a934532ca1ee401d135a5d5917619b62db9711da859ce47f987R64-R67  
not sure what's correct really.

Reply:
why do we need those parseInt() 

can we get rid of those parseInt and number/string checks? those are primary keys and we should know what type it is and be able to use the correct type?

Reply:
I think this came when switching to TS. Once account/allowance is switched let's have a look if we can assume that `id` is always a string and create a PR.